repos/spack_repo/builtin/packages/sedacs/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from spack_repo.builtin.build_systems.python import PythonPackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+
+
+class Sedacs(PythonPackage, CudaPackage):
+    """Scalable Ecosystem, Driver, and Analyzer for Complex Chemistry Simulations (SEDACS) enables massively parallel atomistic simulations that can 
+    seamlessly integrate with a diverse range of available and emerging quantum chemistry codes at different levels of theory.
+
+    Supporting ab initio, semiempirical quantum mechanics, and coarse-grained flexible charge equilibration models, this is a unified 
+    framework to simulate and analyze the quantum molecular dynamics of complex chemical systems and materials.
+    """
+    
+    homepage = "https://github.com/lanl/sedacs"
+    git = "https://github.com/lanl/sedacs.git"
+
+    version('main', branch='main')
+
+    maintainers("finkeljos", "cnegre")
+
+    tags = ["lanl"]
+
+    license("BSD 3-Clause")
+
+    # python dependencies
+    depends_on('python@3.9:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-scikit-build-core', type='build')
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-torch', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
```

Comment:
Based on `pyproject.toml`, these should have minimum versions:

```suggestion
    depends_on('py-numpy@1:', type=('build', 'run'))
    depends_on('py-torch@2:', type=('build', 'run'))
    depends_on('py-scipy@1:', type=('build', 'run'))
```

 Though `py-torch` is the only package that lists older version so technically requires the minimum version.